BraveShared/Extensions/StringExtensions.swift

```diff
@@ -14,4 +14,18 @@ extension String {
         }
         return nil
     }
+    
+    /// Obtain a list of words in a given string
+    public var words: [String] {
+        var words: [String] = []
+        enumerateSubstrings(
+            in: startIndex..<endIndex,
+            options: .byWords
```

Comment:
This solves the previous special characters trim right? the `\, \\ \r` and others

Reply:
Yes, it strips all punctuation that is not part of the word itself (i.e. an apostrophe), I could add more tests including them, one already has a `\r` in it

Reply:
It's ok, no need to test apple's built in method so hard, the pr is good to go